src/ast/semantic_analyser.cpp

```diff
@@ -836,6 +836,12 @@ void SemanticAnalyser::visit(Call &call)
             check_arg(call, Type::integer, 1, true);
           if (call.vargs->size() > 2)
             check_arg(call, Type::integer, 2, true);
+          if (map.type.IsStatsTy() && call.vargs->size() > 1)
+          {
+            ERR("Arguments top and div are disallowed when printing stats maps "
+                "because of the ambiguity.",
```

Comment:
Strange to say "disallowed" when it's actually allowed. Maybe the following wording?

```
print()'s top and div arguments are ignored when used on stats() maps
```

Reply:
Eh, I thought we would use `error` instead of `warning` so I changed from 'ignored' to 'disallow'. Sorry about the mess.